tiledb/sm/query/readers/sparse_global_order_reader.cc

```diff
@@ -1045,19 +1045,20 @@ SparseGlobalOrderReader<BitmapType>::merge_result_cell_slabs(
         }
       }
 
-      update_frag_idx(tile, to_process.pos_ + 1);
-
       // For no dups, we just remove the cells from the queue as the one with
       // the higher timestamp is already in `to_process` and will be processed
       // below. For dups, `to_process` was already added above, replace it with
```

Comment:
Is comment still accurate, '...was already added above', or was that being done by the update_frag_idx() that's now below?

Reply:
It is.